libfreerdp/core/channels.c

```diff
@@ -94,7 +94,7 @@ BOOL freerdp_channel_send(rdpRdp* rdp, UINT16 channelId, const BYTE* data, int s
 			flags |= CHANNEL_FLAG_LAST;
 		}
 
-		if ((channel->options & CHANNEL_OPTION_SHOW_PROTOCOL))
+		if (!rdp->settings->ServerMode && (channel->options & CHANNEL_OPTION_SHOW_PROTOCOL))
```

Comment:
need to check this, any hint what caused this change?

Reply:
Ok, maybe add a reference to `2.2.1.3.4.1 Channel Definition Structure (CHANNEL_DEF)`